spring-cloud-skipper-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -189,6 +189,28 @@ include::{snippets}/package-metadata-documentation/get-all-package-metadata/http
 
 include::{snippets}/package-metadata-documentation/get-all-package-metadata/response-fields.adoc[]
 
+[[resources-package-search-summary]]
+==== Search summary
+
+A `GET` request will return summary information of a package using the _id_ of the package.
```

Comment:
Maybe this isn't a valid description as the summary shows list of all the package metadata with only few fields included in the summary (not using the _id_ of the package).

Reply:
can you remove 'using the id of the package' on merge?

Reply:
ok, I will update something like this: `GET request will return the list of available package metadata with the summary information of each package`